woocommerce/woocommerce-android
# Fix shipping class not set since shipping class name is not found

Fixes #2161. This PR adds logic to display the shipping class name assigned to a product.

~**Note that this [PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/issues/1528) must be merged and the hash updated before this PR can be merged.**~

#### Screenshots
<img src="https://user-images.githubusercontent.com/22608780/78121257-ccb2b880-7428-11ea-9c4e-dec264cbd881.png" width="300" />. <img src="https://user-images.githubusercontent.com/22608780/78121248-cae8f500-7428-11ea-9d1c-1e6de6b5905b.png" width="300" />

#### To test
- Click on a product from the products tab that has shipping class already assigned to it.
- Notice the shipping class name is displayed.
- Click on the Shipping section and update the shipping class.
- Click on `Done`.
- Notice that the shipping class name is updated in both Product detail and in product shipping class.

Update release notes:
